legacy/build.gradle

```diff
@@ -92,6 +92,8 @@ dependencies {
     implementation group: 'org.json', name: 'json', version:'20180813'
     implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.10'
     implementation group: 'org.opensearch', name: 'opensearch', version: "${opensearch_version}"
+    // add geo module as dependency. https://github.com/opensearch-project/OpenSearch/pull/4180/.
```

Comment:
implementation will result into a jar hell. Because OpenSearch will include this jar.

Reply:
Done.